frontend/src/views/LandingPage.vue

```diff
@@ -92,12 +115,60 @@ export default Vue.extend({
     return {
       sessionWrapper: {} as { session: Session },
       startNewSessionOnMounted: false,
+      overAllSessions: 0,
+      overAllAttendees: 0,
+      overAllSessionsFromLastMonth: 0,
+      overAllAttendeesFromLastMonth: 0,
+      currentSessions: 0,
+      currentAttendees: 0,
     };
   },
   created() {
+    this.checkDiveniData(Constants.createDiveniData);
+    this.checkDiveniData(Constants.createDiveniDataFromLastMonth);
+    this.checkDiveniData(Constants.createCurrentDiveniData);
     this.checkAdminCookie();
   },
   methods: {
+    async checkDiveniData(urlData) {
```

Comment:
Extract this to separate analytics service. 
To reduce requests, probably all analytics KPIs could be send within one call.